server/src/test/java/org/elasticsearch/common/network/NetworkUtilsTests.java

```diff
@@ -88,7 +89,14 @@ public void testFilter() throws Exception {
      */
     public void testAddressInterfaceLookup() throws Exception {
         for (NetworkInterface netIf : NetworkUtils.getInterfaces()) {
-            if (!netIf.isUp() || Collections.list(netIf.getInetAddresses()).isEmpty()) {
+            try {
+                if (!netIf.isUp() || Collections.list(netIf.getInetAddresses()).isEmpty()) {
+                    continue;
+                }
+            } catch (SocketException e) {
```

Comment:
I would rather that we gather some reconnaissance on what the failing interface is. I suspect it's a Docker veth interface, but I don't think to date we've collected any evidence of this. I would rather we do something like report the interface and *still* fail the test today (so that we are compelled to look closely at what is going on). It's only when we understand the failure more deeply that we can consider the right solution.

Reply:
Makes sense :) I made it rethrow with the name of the interface included in the error now.

Reply:
How about just ignoring the `veth` interfaces in that test? It seems to me that is the safest bet since Docker could be messing with those in the background?

Reply:
How about just ignoring the `veth` interfaces in that test? It seems to me that is the safest bet since Docker could be messing with those in the background?